### PR TITLE
Create AGP-CS: Develop Aragon Chain

### PR DESCRIPTION
**Pull request by GregTheGreek**
Thread discussing this AGP can be on the [Aragon Forum](https://forum.aragon.org/t/agp-discussion-develop-aragon-chain/1337)

### CODE REVIEW

File: AGPs/AGP-CS.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+Deliverables from this project:
+
+- Publish Aragon Chain spec
+- Develop full-node software that follows the Aragon Chain spec
+- Launch Aragon Chain testnet
+- Launch Aragon Chain mainnet
+
+## Recipient information
+
+TBD
```
Comment (izqui) [inline AGPs/AGP-CS.md]:
```suggestion
```

**Comment by igorbarinov** [comment]
You can use our setup your own EVM chain on DPOS with POA's stack

- POSDAO staking consensus as a DPOS engine https://github.com/poanetwork/posdao-contracts it's implemented and audited. We plan to migrate xDai to it in Q4 2019
- TokenBridge to bridge ANT to a native token of Aragon Chain (like xDai bridge)
- Our new AMB (arbitrary message bridge) to relay any existing and future contracts between Mainnet and your chain https://forum.poa.network/t/how-to-develop-a-cross-blockchain-application-by-using-amb-bridge/2963
- BlockScout as a blockchain explorer
- Staking UI in BlockScout. Get sense of it here https://poa.gitbook.io/xdai-staking/

File: AGPs/AGP-CS.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+# Develop and launch Aragon Chain
+
+## Address of the transfer recipient
+
+TBD
+
+## Amount of the transfer
+
+Upon community review, a better estimate can be made based on the needs and requirements from the community.
```
Comment (john-light) [inline AGPs/AGP-CS.md]: This section will need to be completed with a final amount before the Stage V deadline.

File: AGPs/AGP-CS.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+Status: Stage III
+Track: Finance
+Created: 2019-10-03
+---
+
+# Develop and launch Aragon Chain
+
+## Address of the transfer recipient
+
+TBD
```
Comment (john-light) [inline AGPs/AGP-CS.md]: I recommend that this section be completed with a recipient address before the Stage V deadline.

File: AGPs/AGP-CS.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+## Purpose of the transfer
+
+At the moment, Aragon fully depends on Ethereum mainnet in order to function.
+
+However, Ethereum may not be the best platform for all Aragon use cases. Some Aragon organizations may not want to pay the security and composability premium for being on Ethereum.
+
+On top of this, we are seeing increased development and user experience costs [resulting from changes in the Ethereum protocol that break Aragon organizations](https://www.coindesk.com/ethereums-istanbul-upgrade-will-break-680-smart-contracts-on-aragon).
+
+Building a sovereign layer 1 protocol owned and operated by the Aragon community will ensure that we can keep enhancing Aragon, and even introduce protocol changes that make running decentralized organizations easier and cheaper. This way we can offer Aragon organizations another option parallel to Ethereum, with different security, cost and performance trade-offs. The idea is for this chain to be heavily focused and optimized for the Aragon use case, so other heavyweight use cases that are currently clogging the Ethereum network wouldn't necessarily have any incentives to run on the Aragon Chain.
```
Comment (john-light) [inline AGPs/AGP-CS.md]:
```suggestion
Building a sovereign layer 1 protocol owned and operated by the Aragon community will ensure that we can continue enhancing Aragon, and even introduce protocol changes that make running decentralized organizations easier and cheaper. This way we can offer Aragon organizations another option parallel to Ethereum, with different security, cost, and performance trade-offs. The idea is for this chain to be heavily focused and optimized for the Aragon use case, so other heavyweight use cases that are currently clogging the Ethereum network wouldn't necessarily have any incentives to run on the Aragon Chain.
```

File: AGPs/AGP-CS.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+The Aragon Chain will be an EVM compatible Proof of Stake blockchain whose native staking token is derived from ANT (in a similar fashion as the Aragon Court's ANJ).
```
Comment (john-light) [inline AGPs/AGP-CS.md]:
```suggestion
The Aragon Chain will be an EVM-compatible Proof of Stake blockchain whose native staking token is derived from ANT (in a similar fashion as the Aragon Court's ANJ).
```

File: AGPs/AGP-CS.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+Aragon One has worked with ChainSafe Systems on a feasibility study on what the most fitting technology for building the Aragon Chain is. We are now certain of the way forward and it's ready to get into the specification and development phase.
+
+Feasibility report: TBD (to be made public ahead of the AGP phase V transition)
```
Comment (john-light) [inline AGPs/AGP-CS.md]: Noting the need to fill in the blank for this link

File: AGPs/AGP-CS.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+The report goes indepth between the current leading staking chains: Polkadot and Cosmos. Throughout the report comparisons are made based on Aragon specific optimizations, and cross-chain communication.
```
Comment (john-light) [inline AGPs/AGP-CS.md]:
```suggestion
The report goes in-depth between the current leading staking chains: Polkadot and Cosmos. Throughout the report comparisons are made based on Aragon-specific optimizations, and cross-chain communication.
```

File: AGPs/AGP-CS.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+# Develop and launch Aragon Chain
+
+## Address of the transfer recipient
+
+0xfc6eBB0F689674b18f90dA73658072155e98ea5B
+
+## Amount of the transfer
+
+A total of $x, will be allocated to the Aragon Chain. Invoices will be sent to the Aragon Association, and compensation will be at their discretion.
```
Comment (izqui) [inline AGPs/AGP-CS.md]: Please don't forget to change `x` for the final amount before the proposal needs to be frozen

**Comment by GregTheGreek** [comment]
Please nice to stage v

**Comment by GregTheGreek** [comment]
> You can use our setup for your own EVM chain on DPOS with POA's stack
> 
> * POSDAO staking consensus as a DPOS engine https://github.com/poanetwork/posdao-contracts it's implemented and audited. We plan to migrate xDai to it in Q4 2019
> * TokenBridge to bridge ANT to a native token of Aragon Chain (like xDai bridge)
> * Our new AMB (arbitrary message bridge) to relay any existing and future contracts between Mainnet and your chain https://forum.poa.network/t/how-to-develop-a-cross-blockchain-application-by-using-amb-bridge/2963
> * BlockScout as a blockchain explorer
> * Staking UI in BlockScout. Get a sense of it here https://poa.gitbook.io/xdai-staking/

Porting to forum, since this issue is closed.

**Comment by GregTheGreek** [comment]
@igorbarinov response has been added [here](https://forum.aragon.org/t/agp-discussion-develop-aragon-chain/1337/11?u=gregthegreek)

**Pull request merged**